composer.json

```diff
@@ -12,26 +12,24 @@
         "php": ">=8.1",
         "ext-mbstring": "*",
         "doctrine/collections": "^1.8",
+        "spiral-packages/symfony-serializer": "^2.0",
         "spiral/cycle-bridge": "^2.1",
-        "spiral/framework": "^3.2",
+        "spiral/framework": "^3.7",
         "spiral/nyholm-bridge": "^1.3",
-        "spiral/roadrunner-bridge": "^2.1",
-        "spiral/sapi-bridge": "^1.0.1"
```

Comment:
Are you sure that sapi-bridge is not needed there? It helps to run Spiral under the nginx